### ISSUE TITLE
Allow users to edit the name "Anonymous .*"

### ISSUE THREAD

**Issue by b1f6c1c4** [jupyterlab/jupyterlab#14327]
### Problem

* I'm always frustrated when jupyterlab automatically generates a random name like `Anonymous .*` when using `--collaborative` feature, especially when YOU CANNOT CHANGE IT!!!

### Proposed Solution

Clicking on the small `A.` initial at the upper right corner, or clicking on the third (`ui-components:users`) - OMG there is NO tooltip - tab on the very left - shall prompt the user to change the name.

### Additional context

TBH, all sane collaboration requires one to properly identify theirself under whatever name they want - instead of forcing them to use a name that is randomly drawn from the moons of jupyter.

**Comment by JasonWeill** [comment]
Thanks for opening this issue! If a server component is needed, we may also need to do work in [jupyter_collaboration](https://github.com/jupyterlab/jupyter_collaboration/).

**Comment by ellisonbg** [comment]
In the issue that @JasonWeill linked to above, we are proposing to use `os.getlogin()` for the username when not in collaborative mode.

**Comment by MMesch** [comment]
@brichet the triaging agent has identified this issue as potentially completed. Given the merged PRs that are referenced above that may be true but I'm unsure whether those are a complete solution to this problem. Could you leave a short comment here what is still left to be done or otherwise close the issue as completed?

**Comment by brichet** [comment]
Yes, this is issue is partially fixed as it is now possible to update the user name, display name, etc. after https://github.com/jupyter-server/jupyter_server/issues/1514 and https://github.com/jupyterlab/jupyter-collaboration/pull/482
This update can be performed from the side panel only, not from the "_the small A. initial at the upper right corner_" as suggested in the description.
